docs/reference/search/field-stats.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[search-field-stats]]
+== Field stats API
+
+experimental[]
+
+The field stats api allows to easily figure certain statistical properties without a field without executing a search, but
```

Comment:
Maybe reword to:
`...allows one to find statistical properties of a field without...`